webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -148,79 +166,102 @@
         </div>
       </div>
 
-      <!-- TODO: migrate and test once contact is done
       <div class="col-sm-8 detail-actions right-pane" *ngIf="currentTab === 'contacts'">
-        <div class="actions dropup" *ngIf="showActionBar && selectedContactDoc && !loadingContent && ('contacts.detail' | isState)">
-
+        <div class="actions dropup" *ngIf="showActionBar && selectedContactDoc && !loadingContent && snapshotData?.name === 'contacts.detail'">
           <span>
             <a class="mm-icon mm-icon-inverse mm-icon-caption dropdown-toggle" data-toggle="dropdown" [ngClass]="{'mm-icon-disabled': (!actionBar?.right?.relevantForms || !actionBar?.right?.relevantForms?.length)}">
               <span class="fa fa-plus"></span>
               <p translate>action.report.add</p>
             </a>
-            <ul class="dropdown-menu mm-dropdown-menu with-icon" *ngIf="actionBar?.right?.relevantForms && actionBar?.right?.relevantForms?.length">
-              TODO: Order actionBar.right.relevantForms by 'title'
-              <li *ngFor="let form of actionBar.right.relevantForms; trackBy: form.code">
-                <a show-muted-modal to-state="contacts.report" to-state-params="{ id: selectedContactDoc._id, formId: form.code }">
+            <ul id="relevant-contacts-form" class="dropdown-menu mm-dropdown-menu with-icon" *ngIf="actionBar?.right?.relevantForms && actionBar?.right?.relevantForms?.length">
```

Comment:
Does this need the new id attribute?

Reply:
I added it to facilitate the debugging of html